WebDevStudios/wd_s
# Switch to evilmartians lefthook

Closes #978

### DESCRIPTION

- Updated lefthook to evilmartians version which supports Apple Silicon (M1/M2) - and the latest lefthook fixes issues with spaces in the path 💪

### SCREENSHOTS

<img width="603" alt="Screen Shot 2023-01-10 at 9 53 18 AM" src="https://user-images.githubusercontent.com/630830/211599037-4fc8f8ae-471b-4d15-8e47-3265eba3040b.png">

### OTHER


### STEPS TO VERIFY

1. Check out this branch
2. `nvm use && npm install && composer install` in the theme directory
3. Save a file with a formatting error and try to commit it
4. Verify lefthook prevents the commit.

### DOCUMENTATION

Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?
